### PR TITLE
weechat connect guide

### PR DESCRIPTION
**Pull request by webframp**
I gave this alternative console client a try and figured I might as well write up how I got it connected with IRCRelay

### CODE REVIEW

File: guides/weechat/guide.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+## WeeChat IRCRelay Set-up Guide
+
+WeeChat is a console based IRC client similar to irssi. It is fast,
+lightweight and scriptable in several languages.
+
+### Get the Client
+
+First, you'll need do download and install [weechat](http://www.weechat.org/). If you're on OS X it's available using homebrew:
```
Comment (pearkes) [inline guides/weechat/guide.md]: s/do/to

**Comment by pearkes** [comment]
Awesome, thank you! Just made one comment on a typo I noticed, otherwise, looks good!

**Comment by webframp** [comment]
good catch, fixed

**Comment by pearkes** [comment]
Shweet, appreciate it!

**Pull request merged**